packages/client/src/object/convertWireToOsdkObjects.ts

```diff
@@ -112,14 +118,55 @@ const protoConverterCache = createCache(
 
 const isAfterFeb2024OrNewApis = false;
 
-/**
- * @param objs the objects to be converted, the contents of this array will be mutated
+/*
+ * Currently loses all information about the interface. Future PRs will improve
+ * FIXME
  */
-export async function convertWireToOsdkObjectsInPlace(
+export async function convertWireToOsdkInterfaceInPlace(
```

Comment:
This whole method feels pretty duplicative of `convertWireToOsdkObjectsInPlace` below. I'm kind of surprised that loading from an interface isn't a call to that, followed by an extra fixup step to do the `$as(Interface)` cast.

Reply:
This is actually what happened next in the chain when $as was implemented